### ISSUE TITLE
[SW-Content] Review and Copy edit through prototype of search and appoint rep from eBenefits [CE]

### ISSUE THREAD

**Issue by SgardnerPMP** [department-of-veterans-affairs/va.gov-team#37516]
## Resources
#31230

## TASKS
Review prototype for incorporated edits and edits that were not incorporated
- [x] Review and document in this ticket additional changes still needed.
- [x] Compare and contrast to recommendations made in ticket #31230

## Acceptance Criteria
- [x] Recommendations are documented in this ticket for changes
- [x] Recommend copy edits

**Comment by RLHecht** [comment]
Edits to the prototype:

Make apostrophes curly throughout.
Change to MyVA411 throughout (the y should be lowercase)

**Introduction not signed in page:**
- Remove hyphen in: When you're signed-in to your VA.gov account
- Remove comma in the following sentence: You can save your request in progress, and come back later to finish filling it out.

**Rep type page:**
- Change question per Beth's comments to: What kind of accredited representative are you looking for?
- Second option should be "Representative in a Veterans Service Organization (VSO)" to match the language on the signed in page
- Lowercase lawyer

**Rep select page:**
- All the cards list Veteran Service Organization but it should be Veterans (plural)

**21-22/a File info page:**
- Change to: Your service number (if applicable) (lowercase service and the first word in parentheses)
- Change to: Your insurance number, including the letters in front (remove parentheses, add comma, and lowercase "including")

**Treatment Records page:**
- Change question per Beth's comments to: Choose which treatment records you authorize your accredited representative to access" and delete repetitive text underneath
- Remove bolding from all per Beth's comments
- Change sentence per Beth's comments: You can continue without authorizing us to release any of these treatment records.

**Review page:**
- Change header per Beth's comments to: Your selected accredited representative
- Change first accordion per Beth's comments to: Your selected accredited representative's information 

**Rep Where Signed In page:** 
- Correct spelling of representative in question: Where would you like to search for an accredited representative?

**Applicant Info Signed In page:**
- Remove extra content in note and change text to the following: Note: ~~If you need~~ If you need to update your personal information, please call us at 800-827-1000. We're here Monday through Friday, 8:00 a.m. and 9:00 p.m. ET.

**Review Rep Open page:**
- Type should be Veterans Service Organization

**Confirmation page:**
- Change alert header per Beth's comments to: Download your completed form
- Change button label per Beth's comments to: Download your form

**View Rep page:**
- In dropdown menu, change to Veteran Readiness and Employment (VR&E)
- Also in dropdown menu it looks like there is an extra space before each one except for Financial support

**View Rep EBN page:**
- Same notes as above for dropdown menu
- Change link language to: Sign in to eBenefits to make changes

In reading back through Beth's comments there are some screens and text she mentions to put changes into that I'm not seeing in the current prototype:

**Screen 1**
H1: Change "We found no matches" to "We didn't find a match"
Text: You can go back to try your search again. Try entering a different kind of representative or a different location.
CTA: Go back to search again

On **Screen 2** I don't see the CTA: Find an accredited representative

On **Screen 7** I don't see the following:
Change second bolded header to: "If the accredited representative you want is listed here"
Change text to: Contact the accredited representative you want to use to find out if they're available. 
Change "Let VA know" to "Let us know"
Please submit your form only after an accredited representative has agreed to help you.
If you'd like to appoint a VSO, fill out an Appointment of Veterans Service Organization as Claimant's Representative (VA Form 21-22).
Download VA Form 21-22 (PDF)

If you'd like to appoint an attorney or claims agent, fill out an Appointment of Individual as Claimant's Representative (VA Form 21-22a).
Download VA Form 21-22a (PDF)

Change text to: Mail your completed form to your nearest VA regional office.
Find a VA regional office near you

**Screen 9**: I don't see this screen at all with the following text:
Change header to: "We didn't find a match"
Text: You can go back to try your search again. Try entering a different kind of representative or a different location.
CTA: Go back to search again

**Screen 15:** I don't see where this is:
Change Step 1 of 6 question to: Which accredited representative do you want to help you?

**Screen 19:** I don't see where this is
Make "Here is" a contraction: "Here's the information we have on file for you."

Under "What if my information is wrong?"
Switch the order to correspond with the order of the accordion links, shorten each H3, and use this edited text:

Personal information
If you're enrolled in VA health care, contact your nearest VA medical center to update your personal information.
[Find your nearest VA medical center]
If you're not enrolled in VA health care, but you receive VA benefits, contact your nearest VA regional office to update your personal information.
[Find your nearest VA regional office]

Contact information
Go to your VA.gov profile to update your contact information.
[Go to your profile]

**Comment by bethpottsVADEPO** [comment]
@RLHecht Thanks for your editing and cross checking against the other ticket!

**Comment by RLHecht** [comment]
@candy-gcio I used this ticket to go back through @bethpottsVADEPO comments in the original intake ticket for this work to note any changes that weren't made. Since she had so many in different comments we just wanted to make sure they were all incorporated. I listed what's still needed above. There were also a few screens that Beth mentioned or text in certain screens that I couldn't find in the prototype. I also noted those above. It would be good to make as many changes as you can ahead of the midpoint review so that it goes more smoothly, but I also understand that there isn't a ton of time. If you have any questions about any of my notes please let me know!

**Comment by candy-gcio** [comment]
Hi, @RLHecht - Thanks so much for your comments. I've updated everything I can in the top section of your comments. I'll note the two exceptions with comments/questions below. 

For the updates starting with "Screen 1", all of those comments referred to the very first version in sketch. There were significant changes to the workflow, so I started a "[version 2](https://www.sketch.com/s/6fcfb0c5-8c2d-4231-8b1b-c423e75bf5e2/p/47673127-1304-4E6D-8CBE-C8A160E00A55)" in sketch. At one point, all of those comments were addressed for the screens that still existed, but the screen numbers didn't match up any more. I know we got a few things out of sync when we moved to UXPin. I think you caught all of them in your review, but I'm going back through all three versions to check. 


> Make apostrophes curly throughout.

This is difficult to fix in uxpin, but will be correct in code.


> Review page:
>  - Change header per Beth's comments to: Your selected accredited representative

Are you referring to the H1 here? If so, I think this has to be the same as the rest of the form headers.


cc: @sporeboy, @jason-gcio, @bethpottsVADEPO
